crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/type_alias.py

```diff
@@ -36,3 +47,42 @@
     [T] = T
 type X[T] \
     = T
+
+# type leading comment
+type X = (
```

Comment:
```suggestion
type X = (  # trailing parentheses comment
```
that's one of those that often creates problems